dist/opt/flight/etc/banner/banner.d/10-tips.sh

```diff
@@ -25,6 +25,9 @@
 # https://github.com/openflighthpc/flight-starter
 #==============================================================================
 (
+  if [[ ${flight_STARTER_tips:-enabled} != "enabled" ]] ; then
+    exit
```

Comment:
I think this should be a `return` rather than an `exit` as these files are `source`d rather than executed -- so a sourced `exit` will exit the sourcer, which I don't think is intended, and would prevent `20-status.sh` from running, for example.

Might be wrong though!

Reply:
Seems to work for me but if it's better practice then I shall adapt it.

Evidence: 
```
stu@furugoori ~$ ssh 10.151.15.211                                                                                                                                    255 ↵
Warning: Permanently added '10.151.15.211' (ED25519) to the list of known hosts.
Last login: Wed Jan 17 10:18:44 2024 from 10.151.0.54
  'o`
 'ooo`                                Welcome to your cluster
 `oooo`
  `oooo`         'o`                       Flight Solo 2023.6
    `ooooo`  `ooooo                  Based on Rocky Linux 9.2
       `oooo:oooo`
          `v  -[ alces flight ]-

TIPS:

'flight help'           - get help on available commands
'flight set'            - change login defaults (see 'flight info' for details)
'flight env'            - manage software package ecosystems
'flight desktop'        - manage interactive GUI desktop sessions

CLUSTER STATUS:

Hunted Nodes
  Parsed:    1    Buffer:    2

Configured Nodes
  Completed:    1    Applying:    1    Failed:    0

[flight@prompt-experiment1 ~]$ flight set tips off
flight set: Flight Solo prompt tips disabled.
[flight@prompt-experiment1 ~]$
logout
Connection to 10.151.15.211 closed.
stu@furugoori ~$ ssh 10.151.15.211
Warning: Permanently added '10.151.15.211' (ED25519) to the list of known hosts.
Last login: Wed Jan 17 10:32:19 2024 from 10.151.0.1
  'o`
 'ooo`                                Welcome to your cluster
 `oooo`
  `oooo`         'o`                       Flight Solo 2023.6
    `ooooo`  `ooooo                  Based on Rocky Linux 9.2
       `oooo:oooo`
          `v  -[ alces flight ]-

CLUSTER STATUS:

Hunted Nodes
  Parsed:    1    Buffer:    2

Configured Nodes
  Completed:    1    Applying:    1    Failed:    0

[flight@prompt-experiment1 ~]$
```